amqprs/src/api/channel/basic.rs

```diff
@@ -126,6 +126,14 @@ impl BasicConsumeArguments {
         /// Chainable setter method.
         no_local, bool
     }
+    impl_chainable_alias_setter! {
+        /// Chainable setter method.
+        auto_ack, no_ack, bool
+    }
+    pub fn manual_ack(&mut self, value: bool) -> &mut Self {
+        self.no_ack = !value;
+        self
+    }
     impl_chainable_setter! {
         /// Chainable setter method.
         no_ack, bool
```

Comment:
good to deprecated no_ack method in favor of auto_ack.

Reply:
Then Clippy will warn about all the uses in the library itself.

Reply:
Deprecating the builder method should be fine, though